### ISSUE TITLE
Intake request from IVC Forms - 10-7959f-2 (FMP claim cover sheet)

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#78275]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: IVC Forms
- **OCTO product owner:** Premal Shah
- **Product name:** 10-7959f-2 claim cover sheet
- **Product manager:**  Mary Wang
- **Slack channel:** ivc-forms
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** Rachael Penfil
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [x] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Our team will be digitizing the claim cover sheet form for the Foreign Medical Program (10-7959f-2). We would like CAIA's support in reviewing updates to content and structure.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

early user flows/wireframes

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** [77082](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/77082)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/foreign-medical-program/10-7959f-1/10-7959f-2%20product%20outline.md)]

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by laurwill** [comment]
Hi @brian-wilke @marywang2 , 
Just documenting here the chat we had in our sync last week about the draft user flows for FMP claim cover sheet:
- Draft flows planned to significantly lengthen the form by adding form questions to capture information from the required supporting documents (itemized billing statements from providers)
- I noted that as a general rule, we don't add questions to the online versions of PDF forms, because we're not allowed to increase the time burden on the Veteran unless there's a very good reason to do so
- The only reasons I can think of that we would consider adding questions/sections to a claim form like this are if doing so would significantly speed up processing or avoid subsequent back and forth with the Veteran, and the PRIA reviewers agree that this benefit is worth the increased up front time burden
- So if you hear from SMEs that the current PDF form is not collecting all the info we need and that this is causing problems after submission, that might be a reason to consider expanding the digitized form flow — if not, I think the most straightforward way to digitize would be to only include the questions/data fields from the PDF version, and continue to rely on the required supporting documents to provide the detailed info about the provider, service, and charges
- Another thing I realized after our sync: It's possible the supporting documents, like the itemized statement from the provider, could be in a language other than English. We have a note on the VA.gov page about providing translation in that scenario. And we don't know if Veterans speak the language of the foreign country where they receive care. So it's very possible that the Veteran filling out this form may not be able to read the billing statement, and that would prevent them from answering detailed questions about the info on the statement.

Also resharing the ["How to file an FMP claim" page ](https://www.va.gov/resources/how-to-file-a-va-foreign-medical-program-claim/)here — we recently migrated this content from the legacy FMP webpages, so FMP SMEs very recently reviewed and approved the info here.

Happy to chat more about this form in our next sync. And thank you again for starting the content conversation so early. Looking forward to working together on this!

**Comment by brian-wilke** [comment]
Thank you for this Laura!

**Comment by brian-wilke** [comment]
Totally in agreement, and sorry if this was confusing to anyone.

We realize providers fill this out, but it would always be designed around
the veteran. There is NO CHANGE there.

**Comment by laurwill** [comment]
Hi @brian-wilke @marywang2 , just documenting updates after our chat in office hours yesterday:
- I confirmed with our content team that claims for reimbursement like this form should use the non-application template outlined here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/74785
- We discussed shifting the question about paying Veteran versus provider earlier in the flow, so that we can ask for the right supporting documents (and possibly for the provider's address?)
- **Next step:** Brian will update the Figma and tag Laura here when it's ready for CAIA to flow in

**Comment by baltes** [comment]
Hi @RLHecht, this is to follow up on our slack conversation [here](https://dsva.slack.com/archives/C01K37HRUAH/p1737759481248779).

Our work on form 10-7959f-2 incomplete, but we need support on a couple of things to begin our staged release.

First, we would need content for a react widget to control static page content in the flipper/feature toggle.

Second, at the beginning of the launch we would need to coordinate with someone on publishing the static page when we're ready to open the form to production traffic.

This can definitely get confusing, so let us know if it's easiest to find time to talk through it.

fysa @andrea-merrill

**Comment by mnorthuis** [comment]
Here is the [IA spec for the FMP claim form](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/foreign-medical-program-claim.md).  Please note that the IA implemented in staging is incorrect, and this doc reflects the correct implementation.  The product team will need to correct the URL and the corresponding breadcrumb to represent the correct placement of this form in the IA.

**Comment by RLHecht** [comment]
@baltes Here's the content for the react widget for the File and FMP claim page. I set it up to show the content that will be displayed if the online form is shown and the content that will be displayed if the online form isn't shown. Once the widget is ready to be put into Drupal I can remove the content that's currently on the page. Let me know if you have questions on this:

[Form 10-7952f widget content.docx](https://github.com/user-attachments/files/18790575/Form.10-7952f.widget.content.docx)

**Comment by mistymg** [comment]
As of May 8, 2025, this was [still paused due to launch-blocked bug](https://dsva.slack.com/archives/C01K37HRUAH/p1746730129212929?thread_ts=1744231925.756219&cid=C01K37HRUAH).

**Comment by MarissaMerkt** [comment]
@mistymg do we have any updates on this? What's the current status?

**Comment by mistymg** [comment]
@MarissaMerkt Technically the only piece left to this is for Lauren to publish a page in Drupal. But there was a launch-blocked bug. I scheduled a message to go out to Andrea Merrill next week asking for an update.

**Comment by mistymg** [comment]
This work on FMP is still indefinitely paused [per Slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1751378715773529?thread_ts=1744231925.756219&cid=C01K37HRUAH).

**Comment by MarissaMerkt** [comment]
Discussed this task during the biweekly meeting with the product team today. Andrea said they should have content for us to review by end of sprint and plan to do usability testing before launch.

**Comment by mistymg** [comment]
Got an [update from PM on Health Apps team](https://dsva.slack.com/archives/C05QKRZGHFS/p1767718394410229?thread_ts=1767635749.177059&cid=C05QKRZGHFS)
- This project isn't yet ready to launch. Anticipated February or very early March.
- If will be an incremental release. There's already a widget in place with approved content.

When this goes live, here's what we'll need to do:
1. In coordination with team's launch, do [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/128984) to publish step-by-step.
2. Once online form is at 100%, do [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78275?issue=department-of-veterans-affairs%7Cva.gov-team%7C126080) to put widget content into Drupal + small copy tweak.

Until this launches, putting epic and subissues on hold.

**Comment by mistymg** [comment]
Tentatively launching 2/11. Andrea said she'd tag me in a launch channel when ready.

**Comment by mistymg** [comment]
Health apps team ready for launch. Had Lauren review and publish the step-by-step page that was needed for this. Next step is once at 100%, we remove widget (ticket already created and attached to this epic).
